src/meshweb/views/index.py

```diff
@@ -16,7 +16,8 @@ def index(request: HttpRequest) -> HttpResponse:
         ],
         ("meshweb/volunteer.png", "Volunteer Tools"): [
             ("/admin", "Admin Panel"),
-            ("/api/v1/geography/whole-mesh.kml", "KML Download"),
+            ("https://raw.githubusercontent.com/nycmeshnet/meshdb/refs/heads/main/sampledata/meshdb.kml", "KML Download"),
```

Comment:
Stale data under with the old link name is confusing.

Reply:
That’s not what this is. Check the contents of that file, it’s a 1kb hyperlink pointer to the latest data

Reply:
Wtf

Reply:
No need to reference github https://github.com/nycmeshnet/meshdb/pull/969